packages/@lwc/template-compiler/src/codegen/codegen.ts

```diff
@@ -94,10 +98,31 @@ export default class CodeGen {
         this.scopeFragmentId = scopeFragmentId;
     }
 
+    createScope() {
```

Comment:
To make this symmetrical with the `popScope` method.

```suggestion
    pushScope() {
```